GitUI/UserControls/RevisionGrid/RevisionDataGridView.cs

```diff
@@ -606,6 +606,16 @@ public override void Refresh()
             foreach (var columnProvider in _columnProviders)
             {
                 columnProvider.Refresh(_rowHeight, _visibleRowRange);
+
+                if (FindForm() is GitExtensionsForm form)
+                {
+                    string positionName = $"{Name} {columnProvider.Name}";
+                    var position = form.LookupWindowPosition(positionName);
+                    if (position != null)
+                    {
+                        columnProvider.Column.Width = position.Rect.Width;
```

Comment:
the first column (`GraphColumnProvider`) is dynamically calculated, are we sure it won't introduce another regression?